domain-ee/ee-ep-merge-app/src/python_src/schema/merge_job.py

```diff
@@ -31,7 +31,7 @@ class JobState(StrEnum):
 
     @classmethod
     def incomplete_states(cls):
```

Comment:
suggestion: this may be a good place for a type hint:
```suggestion
    def incomplete_states(cls) -> list[str]:
```